VideoAPI/Video.API/Swagger/AuthResponsesOperationFilter.cs

```diff
@@ -16,7 +16,10 @@ public void Apply(OpenApiOperation operation, OperationFilterContext context)
                 .Any(filter => filter is IAllowAnonymousFilter);
             
             if (!isAuthorized || allowAnonymous) return;
-            operation.Responses.Add("401", new OpenApiResponse { Description = "Unauthorized" });
+            if (!operation.Responses.ContainsKey("401"))
```

Comment:
we could use enum.. but value (int) would again need to be converted to string i think or it would return "Unauthorized"
`((int)HttpStatusCode.Unauthorized).ToString()`